app/models/audit_results.rb

```diff
@@ -41,13 +41,12 @@ class AuditResults
                              '%{db_obj_name} db version (%{db_obj_version})',
     ACTUAL_VERS_LT_DB_OBJ => 'actual version (%{actual_version}) less than ' \
                              '%{db_obj_name} db version (%{db_obj_version}); ERROR!',
-    CM_PO_VERSION_MISMATCH => 'CompleteMoab online Moab version %{cm_version} ' \
```

Comment:
Will getting rid of these be problematic for any results / statuses stored in DB?

Reply:
the statuses in the db are an enum in the models, so I'm pretty sure the db stores them as integers and the descriptions are syntactic sugar for coding.

The AuditResults statuses are just constants we created -- they are not in the database.  We store the result strings.

I will double check both of these.

Reply:
From rails console on stage, after this branch was deployed:

```
3.0.0 :002 > MoabRecord.moab_on_storage_not_found.count
 => 7   # implies the migration didn't need a migration of this data, or this would have been 0
```

```
class AuditResults  # note it is not an ActiveRecord model
  ACTUAL_VERS_GT_DB_OBJ = :actual_vers_gt_db_obj
  ACTUAL_VERS_LT_DB_OBJ = :actual_vers_lt_db_obj
```